CONTRIBUTING.md

```diff
@@ -160,174 +204,115 @@ information treat the first 80 characters as a title. Even Github itself does
+continuous integration tasks should pass and the build should be green.
 
-```html
-<!-- full syntax -->
-<a v-on:click="doSomething"> ... </a>
+You also need to sign the raiden project CLA (Contributor License
+Agreement). Our CLA bot will help you with that after you created a pull
+request. If you or your employer do not hold the whole copyright of the
+authorship submitted we can not accept your contribution.
 
-<!-- shorthand -->
-<a @click="doSomething"> ... </a>
-```
+For frequent contributors with write access to the repository we have a set of rules on pull requests to signal to our colleagues what their current state is:
 
-```html
-<!-- full syntax -->
-<template v-slot:append> ... </template>
+- **Ready for review**: Means that you consider your work done and it is currently waiting for a review
+- **Draft**: Is considered work in progress and it is not yet ready for a review
 
-<!-- shorthand -->
-<a #append> ... </a>
-```
+#### Getting it reviewed
 
-##### Vue components
+It is the responsibility of the author to ask for at least one person for a review. The person should know the area of the code being changed. If the chosen reviewer does not feel confident in the review, they can then ask for someone else to additionally look at the code.
 
-When using custom vue components in templates make sure to use the kebab case name of the components.
+All the developers in the team should review [open pull requests](https://github.com/raiden-network/light-client/pulls) frequently.
 
-Don't do this:
+We have a lot of tools that automatically check the quality of the code (eslint, prettier). All these are automatically ran by the CI. Therefore, fixes related to linting are usually not included in reviews.
 
-```html
-<template>
-  <v-container>
-    <NoValidProvider v-if="!providerDetected" />
-    <UserDenied v-else-if="userDenied" />
-    <WalletCore v-else />
-  </v-container>
-</template>
-```
+Additionally, reviewers should **not be nitpicky** about the suggested changes they ask from the author. If something is indeed nitpicky then the reviewer is encouraged to state it beforehand. Example:
 
-The for consistensy you should use:
+> nitpick: I don't really think XYZ makes sense here. If possible it would be nice to have it changed to KLM
 
-```html
-<template>
-  <v-container>
-    <no-valid-provider v-if="!providerDetected" />
-    <user-denied v-else-if="userDenied" />
-    <wallet-core v-else />
-  </v-container>
-</template>
-```
+The author of the pull request can choose to implement the feedback or to ignore it.
 
-##### CSS naming scheme
+Authors should make pull request reviews easier. Make them **as small as possible** .
 
-For CSS naming we try to follow the [BEM](http://getbem.com/) methodology. Along with the naming,
-you should try to scope the styles in their respective components.
+When a reviewer starts a review, he should write a comment in the pull request stating he is doing so. For example:
```

Comment:
We can probably get rid of this, since we never do this anyway?

Reply:
Maybe we should start doing this instead of removing the rule :)